core/blockchain.go

```diff
@@ -238,7 +248,43 @@ func ExecuteBlockEphemerallyForBSC(
 		return nil, fmt.Errorf("writing changesets for block %d failed: %w", header.Number.Uint64(), err)
 	}
 
-	return receipts, nil
+	// these parts were not present before. So, maybe we don't need them?
```

Comment:
this is not needed

Reply:
well, without them we can't have a proper ReceiptForStorage and LogsHash. Do you think that would be okay?